volcli/volcli.go

```diff
@@ -726,6 +727,18 @@ func volumeSnapshotCopy(ctx *cli.Context) (bool, error) {
 		return false, errored.Errorf("Volume %v Response Status Code was %d, not 200", qualifiedVolume, resp.StatusCode)
 	}
 
+	content, err = ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return false, errored.New("Reading body processing response").Combine(err)
+	}
+
+	vol := &config.Volume{}
+	if err := json.Unmarshal(content, vol); err != nil {
+		return false, errors.UnmarshalVolume.Combine(err)
+	}
+
+	fmt.Println(strings.Join([]string{vol.PolicyName, vol.VolumeName}, "/"))
```

Comment:
Yes, this is unmarshalled data from the API server which we print in the CLI. The data itself came from the API server which they will query directly.